requirements.txt

```diff
@@ -1,2 +1,2 @@
-PyYAML
-websocket-client
\ No newline at end of file
+PyYAML==6.0
+websocket-client==1.4.2
```

Comment:
I've thought about this further, and I think that given that `ops` is a library and not an applicable, we shouldn't pin exact dependencies. For rationale, see [this SO answer](https://stackoverflow.com/a/44938662/68707). I think we should only pin to the major (compatible) version. So:

```
PyYAML==6.*
websocket-client==1.*
```

We can reevaluate later if needed, but I think that's safe, and an improvement on what we have (no pinning).

For `requirements-dev.txt` we should still pin exact versions, as those are only used locally.